commitlint.config.js

```diff
@@ -0,0 +1 @@
+module.exports = { extends: ["@commitlint/config-conventional"] };
```

Comment:
Were you unable to move this into configs? I'm not as familiar with our node runtimes on this point